app/lib/submission_builder/ty2024/states/md/documents/md502_cr.rb

```diff
@@ -10,13 +10,15 @@ class Md502Cr < SubmissionBuilder::Document
 
             def document
               build_xml_doc("Form502CR", documentId: "Form502CR") do |xml|
-                if calculated_fields.fetch(:MD502_DEDUCTION_METHOD) == "S"
+                if deduction_method_is_standard || calculated_fields.fetch(:MD502CR_PART_CC_LINE_7)&.positive?
```

Comment:
change to &.present

Reply:
if `:MD502CR_PART_CC_LINE_7` is `0` --> do we want to include?
if negative value --> we do want to include?

include as long as _some_ value is there